examples/ed25519/tests/test_ed25519.rs

```diff
@@ -0,0 +1,88 @@
+use hacspec_dev::prelude::*;
+use hacspec_ed25519::*;
+use hacspec_lib::*;
+
+create_test_vectors!(
+    Ed25519TestVector,
+    message: String,
+    pub_key: String,
+    signature: String
+);
+
+// Testing test vectors from https://eprint.iacr.org/2020/1244.pdf
+// https://github.com/novifinancial/ed25519-speccheck
```

Comment:
What about the RFC test vectors? https://datatracker.ietf.org/doc/html/rfc8032#section-7
Could you add them as well?